app/scripts/components/common/blocks/scrollytelling/index.tsx

```diff
@@ -492,3 +469,12 @@ Scrollytelling.propTypes = {
 export function ScrollytellingBlock(props) {
   return <BlockErrorBoundary {...props} childToRender={Scrollytelling} />;
 }
+
+export function reconcileVizDataset(dataset): VizDatasetSuccess {
+  return {
+    status: DatasetStatus.SUCCESS,
+    data: dataset,
+    error: null,
+    settings: dataset.settings
+  };
+}
```

Comment:
💭 : Should this move over to a more general place like `data-utils` file if other components that need visualization type might use this?

Reply:
Yeah, probably it will have to be moved and re-used